examples/bri-3/prisma/migrations/20230323222642_add_merkle_tree/migration.sql

```diff
@@ -0,0 +1,7 @@
+-- CreateTable
```

Comment:
Could you just do one migration instead of three as we do not have any production environment and data to migrate?

Reply:
agree, even if we did, this PR is not merged so no reason to have migrations to add and drop table and so on

Reply:
compressed to a single migration